clients/cobol-lsp-vscode-extension/src/test/suite/lsp.spec.test.ts

```diff
@@ -176,18 +176,20 @@ suite("Integration Test Suite", function () {
     const editor = helper.getEditor("ADSORT.cbl");
     await helper.waitFor(async () => {
       await helper.sleep(100);
-      const result = await vscode.commands.executeCommand<vscode.Location[]>(
+      const result =
+        (await vscode.commands.executeCommand<vscode.Location[]>(
+          "vscode.executeDefinitionProvider",
+          editor.document.uri,
+          pos(58, 36),
+        )) ?? [];
+      return result.length > 0;
```

Comment:
```suggestion
      return result?.length > 0;
```
The same below in the assert.

Reply:
Fixed